tests/utils/test_shortlimitholdem_utils.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from rlcard.games.shortlimitholdem.utils import ShortHand
+
+class TestShortLimitHoldemUtils(unittest.TestCase):
+    ''' ShortLimitHoldem is not much different from regular LimitHoldem. However, in LimitHoldem,
+    aces only count as a high card, while they count as a low and high card in Short.
+    Because of this, the majority of these tests make sure this works.
```

Comment:
Aces should count as either high or low in any holdem variation, if that wasn't the case for limit holdem then it was a bug in their implementation

Reply:
Interesting. I'll fix it up in Limitholdem which should then work for Short as well as No Limit.